qemu/tests/cfg/migrate.cfg

```diff
@@ -87,6 +88,7 @@
                 - read_only_dest:
                     test_type = test_read_only_dest
                     only exec
+                    max_vms = 3
```

Comment:
`virt_test_vm1_guest` is the one we use it for migration, `virt_test_vm2_data_server` is used to have NFS server which doesn't require Hugepages right ?
only the `virt_test_vm1_guest` and its `target` requires Hugepages and max_vms = 2 would suffice ?

Reply:
by default all machines use the same setting, therefor if we chose `hugepages` both the guest and the server will use hugepages. Then the migration will be triggered, therefor you need enough hugepages to run 3 machines simultaneously (but we don't need 4 as the `data_server` is not being migrated).

Reply:
Okay, got it.